src/solutions/components/SolutionCard.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Circle,
+  HStack,
+  Heading,
+  Stack,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import DisplayCard from "components/common/DisplayCard"
+import Image from "next/image"
+
+export type Props = {
+  title: string
+  imageUrl: string
+  description: string
+  bgImageUrl: string
+  onClick?: (data?: any) => void
+  children?: JSX.Element
+}
+
+const SolutionCard = ({
```

Comment:
Not sure why, but this element is not focusable with keyboard navigation, we should look into it imo!